website/api/duk_to_summary.yaml

```diff
@@ -0,0 +1,26 @@
+name: duk_to_summary
+
+proto: |
+  const char *duk_to_summary(duk_context *ctx, duk_idx_t idx);
+
+stack: |
+  [ ... val! ... ] -> [ ... summary! ... ]
+
+summary: |
+  <p>Replace the value at <code>idx</code> with a readable summary of the value
+  in string form.  If <code>idx</code> is invalid, throws an error.</p>
+
```

Comment:
I agree, this was just a quick draft.  It's mostly a placeholder so I don't forget to write the API doc later :)